app/jobs/fs/sync_box_job.rb

```diff
@@ -1,11 +1,27 @@
 module Fs
   class SyncBoxJob < ApplicationJob
-    def perform(box, from: Date.today - 1.week, to: Date.tomorrow)
+    def perform(box, from: Date.yesterday, to: Date.tomorrow, fs_client: FsEnvironment.fs_client, batch_size: 25)
```

Comment:
Default batch size by som dal aspoň 100.

Reply:
Tu predpokladam, ze to nespravi velmi rozdiel, lebo sa bavime o batch_size nad spravami jedeho boxu za 2 dni. Pricom bezne sa za jeden box podavaju max mozno 4 podania mesacne. Neocakavame, ze by nastal pripad viac ako 25 sprav.